testcafeRunner.js

```diff
@@ -16,7 +16,7 @@ createTestcafe('localhost')
 
     return tc.createRunner()
       .src(['**/*ui.test.js'])
-      .browsers('chrome:headless')
+      .browsers('chrome')
```

Comment:
Check your merge, I'm pretty sure Fawad has changed this file

Reply:
reverted back to `.browsers('chrome:headless')`, git should handle the rest